docs/dao-creation.md

```diff
@@ -0,0 +1,117 @@
+# DAO Creation Guide
+
+## Overview
+
+Soroban Governer is a Smart Contract system that allows for the creation and management of Decentralized Autonomous Organizations (DAOs). This guide will walk you through the process of creating a DAO using the Soroban Governor CLI and interacting with it using the Soroban Governor interface.
+
+Soroban Governor is based heavily on OpenZeppelin's Governor Alpha contract. As such, the process of creating and managing a DAO using Soroban Governor is very similar to that of OpenZeppelin's Governor Alpha.
+
+## Deployment
+
+TODO: should we reccomend the CLI for this?
+
+## Initialization
+
+Now that you've deployed your Governor and Votes contracts, you need to initialize them.
+
+The Votes contract can be initialized with the `initialize()` function. This function takes a single argument, `token`, which is the address of the token contract the DAO uses for voting.
```

Comment:
Wording here is a bit confusing.

`Votes` is the token the DAO uses for voting.

Maybe - 
```
This function takes a single argument, `token`, which is the address of the underlying token contract the Votes contract will support.
```